plugin/src/main/resources/io/jenkins/plugins/coverage/model/visualization/dashboard/CoverageColumn/column.jelly

```diff
@@ -1,27 +1,30 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
 
+  <link rel="stylesheet" href="${resURL}/plugin/code-coverage-api/css/column-style.css"/>
+
   <j:set var="coverageText" value="${it.getCoverageText(job)}"/>
   <j:set var="coverageValue" value="${it.getCoverageValue(job)}"/>
 
   <j:choose>
     <j:when test="${coverageValue.isPresent()}">
       <j:set var="displayColors" value="${it.getDisplayColors(job, coverageValue)}"/>
-      <j:set var="lineColor" value="${displayColors.getLineColorAsHex()}"/>
```

Comment:
The intention behind this line is that it is guaranteed that the color of the text is readable on the background color.
With the currently used color palette, this shouldn't be a problem.
I guess this has been removed to make all URLs in the dashboard the same color?
When this line is removed, we have to take care that there are no background colors in use which causes problems with the readability.

Reply:
When we set a color then we should use the predefined color variables of Jenkins. Without the color, the links are using the Jenkins default, which should look good in all themes:

![Bildschirmfoto 2022-05-22 um 22 11 24](https://user-images.githubusercontent.com/503338/169713968-8359a449-acb5-42a3-bcd6-df3bccb15474.png)